### PR TITLE
Update current erc20 library + Remove a doc comment

### PR DESCRIPTION
**Pull request by gRoussac**
Update lib for 

https://github.com/casper-ecosystem/erc20/issues/39

Remove legacy command in comments

https://github.com/casper-ecosystem/erc20/issues/49

### CODE REVIEW

**Review by AlexanderLimonov — APPROVED** [review]
Seems reasonable

File: docs/TUTORIAL.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ make test
 
 # Contract Implementation
 
-In [GitHub](https://github.com/casper-ecosystem/erc20), you will find a library and an [example implementation]((https://github.com/casper-ecosystem/erc20/blob/master/example/erc20-token/src/main.rs)) of the Fungible Token for the Casper Network. This section explains the example contract in more detail.
+In [GitHub](https://github.com/casper-ecosystem/erc20), you will find a library and an [example implementation](<(https://github.com/casper-ecosystem/erc20/blob/master/example/erc20-token/src/main.rs)>) of the Fungible Token for the Casper Network. This section explains the example contract in more detail.
```
Comment (ipopescu) [inline docs/TUTORIAL.md:89]: I suggest removing this double bracket.

```suggestion
In [GitHub](https://github.com/casper-ecosystem/erc20), you will find a library and an [example implementation](https://github.com/casper-ecosystem/erc20/blob/master/example/erc20-token/src/main.rs) of the Fungible Token for the Casper Network. This section explains the example contract in more detail.
```

File: docs/TUTORIAL.md [hunk 2]
```diff
@@ -281,22 +284,26 @@ casper-client put-deploy \
 --secret-key "/home/ubuntu/secret_key.pem" \
 --payment-amount 1000000 \
 --session-path "<machine-path>/erc20/target/wasm32-unknown-unknown/release/erc20_test.wasm"
+--session-arg "name='Token test', symbol='TEST', decimals:u8=10, total_supply:u256=1000"
```
Comment (ipopescu) [inline docs/TUTORIAL.md:287]: Thanks for adding this!

**Review by ipopescu — COMMENTED** [review]
Thanks for the updates, @gRoussac. LGTM! I'll leave the approval to the devs :).

**Review by deuszex — APPROVED** [review]
Looks good, tests run green, lint is good also.

**Pull request merged**